STACpopulator/populator_base.py

```diff
@@ -115,12 +122,12 @@ def ingest(self) -> None:
         for item_name, item_data in self._ingest_pipeline:
             LOGGER.info(f"Creating STAC representation for {item_name}")
             stac_item = self.create_stac_item(item_name, item_data)
-            post_stac_item(self.stac_host, self.collection_id, item_name, stac_item)
-            try:
-                pass
-            except Exception:
-                LOGGER.error(f"Failed adding STAC item {item_name}")
-                self.handle_ingestion_error("Posting Error", item_name, item_data)
+            # post_stac_item(self.stac_host, self.collection_id, item_name, stac_item)
+            # try:
+            #     pass
+            # except Exception:
+            #     LOGGER.error(f"Failed adding STAC item {item_name}")
+            #     self.handle_ingestion_error("Posting Error", item_name, item_data)
```

Comment:
old code to remove?

Reply:
No, we still have to add the posting code. This will be fixed after this PR :)